### PR TITLE
Continue survey of online resources

### PR DESCRIPTION
**Pull request by jngrad**
Description of changes:
- define OER and provide citations
- continue search for resource directories

### CODE REVIEW

**Comment by jngrad** [comment]
@knarrff @hvwaldow This is a follow-up to #39 and #86, starting from the newly split manuscript `survey.md`.

**Comment by CaptainSifff** [comment]
We  now have #111

File: survey.md [hunk 1]
```diff
@@ -168,49 +168,59 @@ engineering community [@SURESOFT2022].
...
-Likewise, the reference work on RSE by Fogel [@Fogel2005] was released in its
-second edition as a living document [@Fogel2017]. The ENCCS instructor
-training manual [@ENCCS2022] is also available as a living document
-[@ENCCSInstructorTraining]. The Carpentry instructor guide [@Wilson2019a]
+The ENCCS instructor training manual [@ENCCS2022] is available as a living document
+[@ENCCSInstructorTraining], and The Carpentry instructor guide [@Wilson2019a]
 is available as living documents in both English [@Wilson2019b] and Spanish [@Wilson2019c].
-From the MIT Computer Science & Artificial Intelligence Laboratory comes an
-unofficial course [The Missing Semester of Your CS
-Education](https://missing.csail.mit.edu/) which covers a lot of basic computer
```
Comment (CaptainSifff) [inline survey.md:184]: wouldn't it make sense, to still have these web resources in the survey web-page?

File: survey.md [hunk 2]
```diff
@@ -168,49 +168,59 @@ engineering community [@SURESOFT2022].
...
+so that the community of trainers and learners can make the most out of it [@Garcia2020].
+Material with rich metadata can be indexed in online databases,
+such as the INTERSECT [@Carver2020] collection [@INTERSECTOnlineResources]
 and the US-RSE collection [@usRSETraining] for all RSE specialisations,
 the ELIXIR [@Gurwitz2020] Training e-Support System (TeSS) platform
 [@Beard2020; @Bacall2023] for bioinformatics and life sciences,
 or the now-defunct Educational Resource Discovery Index (ERuDIte)
 [@VanHorn2017; @Ambite2021] for data science.
 
+FIXME: better integrate this MIT course, maybe move it to the introduction?
```
Comment (CaptainSifff) [inline survey.md:207]: Most likely it requires a bit of restructuring anyway, when we move to a web resource.

File: survey.md [hunk 3]
```diff
@@ -273,6 +283,12 @@ way to clearly label
 - what are the topics required to know to start the course
 This, in part, is due to a missing (metadata) standard to express these dependencies.
 
+FIXME:
```
Comment (CaptainSifff) [inline survey.md:286]: we can put these into separate headings in the web resource.

**Comment by CaptainSifff** [comment]
An awesome amount of new resources! pinging @geyslein .

**Pull request merged**